engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -445,18 +445,17 @@ case class WorkflowExecutionActor(workflowDescriptor: EngineWorkflowDescriptor,
   private def processBypassedScope(jobKey: JobKey, data: WorkflowExecutionActorData): Try[WorkflowExecutionDiff] = {
     Success(
       WorkflowExecutionDiff(
-        executionStoreChanges = Map(jobKey -> ExecutionStatus.Bypassed),
-        valueStoreAdditions = bypassedScopeResults(jobKey)
+        executionStoreChanges = Map.empty, // Map(jobKey -> ExecutionStatus.Bypassed),
+        valueStoreAdditions = Map.empty //bypassedScopeResults(jobKey)
```

Comment:
Is this method still useful then ? The purpose of the `WorkflowExecutionDiff` is to represent changes in the execution store and value store after a key has been processed. So if they're both empty here it's not doing anything ?

Reply:
Yeah I mentioned this in my "look at again for complex cases". FWIW I'm pretty sure the answer is "it can be removed"